tests/cli/workflow/rnafusion/test_cli_rnafusion_compound_commands.py

```diff
@@ -183,7 +183,18 @@ def test_store_available(
     # GIVEN that HermesAPI returns a deliverables output
     mocker.patch.object(HermesApi, "convert_deliverables")
     HermesApi.convert_deliverables.return_value = CGDeliverables(**hermes_deliverables)
-    #
+
+    # Ensure the config is mocked to run compound command
+    Path.mkdir(
+        Path(
+            rnafusion_context.meta_apis["analysis_api"].get_case_config_path(case_id_success)
+        ).parent,
+        exist_ok=True,
+    )
+    Path(rnafusion_context.meta_apis["analysis_api"].get_case_config_path(case_id_success)).touch(
+        exist_ok=True
+    )
```

Comment:
Should this be  a fixture instead for reuse by other RNAFusion CLI cmds and make the test easier to read?